Misc/NEWS.d/next/Core and Builtins/2017-12-22-13-28-07.bpo-32390.QPj083.rst

```diff
@@ -0,0 +1,2 @@
+Correct compiler syntax error on AIX for st.f_fsid addition
+Error message: 1506-280 (S) Function argument assignment between types "unsigned long" and "struct fsid_t" is not allowed.
```

Comment:
I would prefer:

Fix ``os.statvfs(path)`` on AIX where the ``f_fsid`` member of the ``statvfs`` structure is a structure instead of unsigned long. Patch by Michael Felt.

Reply:
I did more research and found a way for typedefs to be the same. Added comments in code and summary in News.

If this is not the way you want it documented, please reply.

Reply:
A comment in a source file with part of a AIX header being copy-pasted in it is not correct, this should be done in the issue instead.

Please start a discussion in the issue.